### PR TITLE
Regenerate files

### PR DESCRIPTION
**Pull request by jamesaoverton**

### CODE REVIEW

File: registry/publications.md [hunk 1]
```diff
@@ -3,17 +3,15 @@ layout: doc
 title: Publications Related to the OBO Foundry
 ---
 
-## Papers about the OBO Foundry
+[The OBO Foundry: coordinated evolution of ontologies to support biomedical data integration](http://www.nature.com/nbt/journal/v25/n11/abs/nbt1346.html)
```
Comment (nlharris) [inline registry/publications.md:6]: Hi Lynn, I'm confused because it looks like you took out the new 2021 paper, but I might be reading the diff wrong?

**Review by nlharris — COMMENTED** [review]
I had a comment about your changes to publications.md.

File: registry/publications.md (hunk 1 above)
Comment (jamesaoverton) [reply registry/publications.md:6]: @nlharris This update pulls in the last 50 commits, including #1634, not just DISDRIV.

Comment (jamesaoverton) [reply registry/publications.md:6]: @nlharris Ahh, you're right, there is a problem. Your #1634 modified `registry/publications.md`, but that's an automatically generated file, and my update overwrote your changes. I'll fix this. Thanks!

Comment (jamesaoverton) [reply registry/publications.md:12]: @beckyjackson fixed this, so I'll mark as 'resolved'.

**Review by nlharris — APPROVED** [review]

**Pull request merged**